src/test/java/me/karakelley/primefactors/PrimeFactorsTest.java

```diff
@@ -45,6 +45,16 @@ public void testFactorsOfNine() {
     assertEquals(list(3, 3), PrimeFactors.calculate(9));
   }
 
+  @Test
+  public void testFactorsOfTwentySeven() {
+    assertEquals(list(3, 3, 3), PrimeFactors.calculate(27));
+  }
+
+  @Test
+  public void testFactorsOfSixHundredTwentyFive() {
+    assertEquals(list(5, 5, 5, 5), PrimeFactors.calculate(625));
+  }
+
```

Comment:
I feel that the tests are similar to each other and you could refactor at one point to have one assertion and some type of matrix that feeds it that has the numbers and expected prime numbers that you could iterate through